docs/tutorial/layers.md

```diff
@@ -486,3 +486,15 @@ The `SLICE` layer is a utility layer that slices an input layer to multiple outp
 #### Mean-Variance Normalization
 
 `MVN`
+
+#### Silence
+
+* LayerType: `SILENCE`
+* CPU implementation: `./src/caffe/layers/silence_layer.cpp`
+* CUDA GPU implementation: `./src/caffe/layers/silence_layer.cu`
+* Sample
+
+	layers { name: "silence" bottom: "input" type: SILENCE }
```

Comment:
Please update to the current proto format by `layers` -> `layer` and `type: SILENCE` -> `type: "Silence"`